_i18n/ru/resources/moneropedia/weblate/ring-size.po

```diff
@@ -48,22 +44,16 @@ msgstr "Основная информация"
 #. type: Plain text
 #: ../_i18n/en/resources/moneropedia/ring-size.md:11
 msgid "Ring size refers to the total number of signers in a @ring-signature. Before release [0.13.0 \"Beryllium Bullet\"]({{ site.baseurl }}/2018/10/11/monero-0.13.0-released.html), it was possible to select an arbitrary number of signers. With release 0.13, it was decided to set this number to 11 to enforce transaction uniformity."
-msgstr ""
-"Размером кольца называется общее количество подписантов, присутствующих в "
-"@кольцевой-подписи. До выхода версии [0.13.0 \"Beryllium Bullet\"]({{ "
-"site.baseurl }}/2018/10/11/monero-0.13.0-released.html) можно было выбрать "
-"любое произвольное количество подписантов. При реализации этой версии "
-"протокола было решено сделать данный параметр фиксированным и равным 11, что "
-"обеспечивало бы единообразие транзакций."
+msgstr "Размером кольца называется общее количество подписантов, присутствующих в @кольцевой-подписи. До выхода версии [0.13.0 \"Beryllium Bullet\"]({{ site.baseurl }}/2018/10/11/monero-0.13.0-released.html) можно было выбрать любое произвольное количество подписантов. При реализации этой версии протокола было решено сделать данный параметр фиксированным и равным 11, что обеспечивало бы единообразие транзакций."
 
 #. type: Plain text
 #: ../_i18n/en/resources/moneropedia/ring-size.md:13
-msgid "`Ring size (11) = foreign outputs (10) + your output (1)`"
+#, fuzzy
+#| msgid "`Ring size (11) = foreign outputs (10) + your output (1)`"
+msgid "`Ring size (16) = foreign outputs (15) + your output (1)`"
 msgstr "`Размер кольца (11) = чужие выходы (10) + ваш выход (1)`"
```

Comment:
```suggestion
msgstr "`Размер кольца (16) = чужие выходы (15) + ваш выход (1)`"
```

Reply:
This should be fixed from weblate, but it's complex because marked fuzzy. I fixed it manually. It will generate conflicts, but i'll deal with them on weblate.